docs/getting-started-guides/docker-multinode.md

```diff
@@ -1 +1,6 @@
-# Stop. This guide has been superseded by [kubeadm](../kubeadm/). This page is only present for historical purposes.
+---
```

Comment:
@zacharysarah - this document was pretty non-standard, so I wasn't sure if I should add the YAML title declaration so it would render the caution as requested, or if there's a better option here - like

```---
sitemap: false
redirect_to:
- "/docs/admin/kubeadm"
---```

Any preference on how this gets updated?

Reply:
Good question! I like the `redirect_to` better. 👍

Reply:
We're trying to stay away from client-side redirects. I'll take a closer look at this later today.

Reply:
@steveperry-53 Good to know. :+1: In that case, there'd be an update required for [_redirects](https://github.com/kubernetes/kubernetes.github.io/blob/master/_redirects#L165)--and it turns out there's already an entry for `docs/getting-started-guides/docker-multinode.md`.

@heckj :wave: No `redirect_to` is needed. Let's wait for @steveperry-53's approval, but otherwise this LGTM.

Reply:
@heckj Do you want the old topic to stay in place and have a link to the new topic? Or do you want the old URL to be automatically redirected to the new URL (user never sees the old topic)?

Reply:
@steveperry-53 I didn't have a firm desire either way - was trying to apply the "make the THIS IS DEPRECATED" setup more visible, and ran into this stand-alone page that I'm guessing is only there due to legacy reasons. There's no content for docker-multinode, so as far as I'm concerned the user could be very safely redirected without ever seeing the topic.

Reply:
@heckj :wave:
> as far as I'm concerned the user could be very safely redirected without ever seeing the topic.

In that case, please add this line to [`_redirects`](https://github.com/kubernetes/kubernetes.github.io/blob/master/_redirects) in alphabetical order:
```
docs/getting-started-guides/centos/centos_manual_config/     /docs/admin/kubeadm/     301
```
I'm guessing you can also discard the caution callout added to `docs/getting-started-guides/centos/centos_manual_config.md`.

Reply:
Thanks y'all - the redirect that I thought was relevant is actually for `docs/getting-started-guides/docker-multinode.md`, and in looking within `_redirects`, it appears to be there already, so I've been screwing around with a file that's not even relevant - at least as I understand the redirect set listings. I'll remove this file and update this PR, rebasing on latest as well...

I think leaving the centos-manual stuff in there makes sense for now and likely shouldn't yet be deprecated and entirely removed, as there's still quite valid information in there - at some point the kubeadm world will replace it entirely...

Reply:
@zacharysarah 👋 could use your review - I think I've addressed your desire with the caution setup and removed the extraneous file now